plugin-server/tests/main/process-event.test.ts

```diff
@@ -1032,6 +1032,44 @@ test('snapshot event stored as session_recording_event', async () => {
     expect(event.snapshot_data).toEqual({ timestamp: 123 })
 })
 
+it('snapshot event not stored if above recordings limit', async () => {
+    await hub.db.postgresQuery('update posthog_team set session_recording_opt_in = $1', [true], 'testRecordings')
+    await hub.db.postgresQuery(
+        `update posthog_organization set usage = '{"recordings": {"usage": 101, "limit": 100}}'`,
```

Comment:
Aside: I think using jsonb for this was a mistake - more columns would be cheaper and would integrate better with django orm, validations, etc.